src/llm/__init__.py

```diff
@@ -18,13 +18,14 @@
 from settings import OPENAI_API_KEY, PROJECT_CONFIG, VERBOSE_LLM
 from sqlalchemy import asc, select
 
+EMBEDDINGS_LLM = OpenAIEmbeddings(openai_api_key=OPENAI_API_KEY)
+PROMPT = PROJECT_CONFIG.get("prompt")
+MODEL_NAME = PROMPT.get("model_name", default="gpt-3.5-turbo")
```

Comment:
this configuration should be in the settings package

https://github.com/talkdai/dialog/blob/main/src/settings.py